components/collection/unlockable/UnlockableLandingMobileBanner.vue

```diff
@@ -7,10 +7,15 @@
         height="42"
         src="/drop/unlockable-pulse-static.svg"
         alt="unlockable icon" />
-      <span>{{ $t('mint.unlockable.mintLive') }}</span>
+      <span>{{ mintStatusText }}</span>
     </div>
-    <nuxt-link class="has-text-weight-bold" :to="DEFAULT_DROP">
-      {{ $t('mint.unlockable.takeMe') }}
+    <nuxt-link class="has-text-weight-bold" :to="to">
```

Comment:
Ts warning here on `:to=to` prop

Reply:
![CleanShot 2023-12-19 at 17 48 35](https://github.com/kodadot/nft-gallery/assets/44554284/7c55ef17-d7ea-455f-8021-dab1e64659c9)